packages/manager/modules/pci/src/projects/project/failover-ips/index.js

```diff
@@ -2,21 +2,26 @@ import angular from 'angular';
 import '@uirouter/angularjs';
 import 'oclazyload';
 
+import redirection from './redirection';
+
 const moduleName = 'ovhManagerPciProjectFailoverIpsLazyLoading';
 
-angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
-  /* @ngInject */ ($stateProvider) => {
-    $stateProvider.state('pci.projects.project.failover-ips.**', {
-      url: '/failover-ips',
-      lazyLoad: ($transition$) => {
-        const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+angular
+  .module(moduleName, ['ui.router', 'oc.lazyLoad'])
+  .config(
+    /* @ngInject */ ($stateProvider) => {
+      $stateProvider.state('pci.projects.project.failover-ips.**', {
```

Comment:
Shouldn't the state be renamed? (Even the whole module?)

Reply:
Renaming state and whole module means lot of refactoring, high impact. I think its better to avoid. What do you think.

Reply:
Hmm yes it could be done in a separate task if there is a lot of refactoring, but that's kind of disturbing of having route named differently than the module.